### ISSUE TITLE
LUA Error on equip "still tradeable" item

### ISSUE THREAD

**Issue by fubaWoW** [KyrosKrane/AnnoyingPopupRemover#55]
```LUA
2x .../Blizzard_StaticPopup_Frame/Mainline/StaticPopup.lua:1822: attempt to index local 'data' (a number value)
[string "@Blizzard_StaticPopup_Frame/Mainline/StaticPopup.lua"]:1822: in function `OnAccept'
[string "@AnnoyingPopupRemover/module_equip_tradable.lua"]:117: in function `?'
[string "@AnnoyingPopupRemover/AnnoyingPopupRemover.lua"]:453: in function <AnnoyingPopupRemover/AnnoyingPopupRemover.lua:452>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `UseContainerItem'
[string "@Blizzard_UIPanels_Game/Mainline/ContainerFrame.lua"]:1422: in function `ContainerFrameItemButton_OnClick'
[string "@Blizzard_UIPanels_Game/Mainline/ContainerFrame.lua"]:1481: in function <...s/Blizzard_UIPanels_Game/Mainline/ContainerFrame.lua:1468>

Locals:
self = <table> {
 button2 = "Cancel"
 exclusive = 1
 button1 = "Okay"
 text = "Performing this action will make this item non-tradeable."
 hideOnEscape = 1
 whileDead = 1
 timeout = 0
}
data = 0
(*temporary) = nil
(*temporary) = "attempt to index local 'data' (a number value)"

```

**Comment by KyrosKrane** [comment]
Oof, this is a new one. What item were you trying to equip? And what was its status, bind-on-equip or warbound-until-equipped?

**Comment by itsSinba** [comment]
Came to report the same error. Happens when equipping raid gear and getting the confirmation dialog that the item will be soulbound. The error pops up then.

**Comment by KyrosKrane** [comment]
Thank you! I'll start investigating this now.

**Comment by fubaWoW** [comment]
> Oof, this is a new one. What item were you trying to equip? And what was its status, bind-on-equip or warbound-until-equipped?

It was BoP item(s) but with the Status to "Trade" it the next 2 Hours with eligible other players.
It was an Item from LFR Bosses
But on Equip the item will lose it's "Tradeable" Status and this Popup triggered the error

**Comment by KyrosKrane** [comment]
This is fixed in v21.4.0-release, which should be available at the usual places in a few minutes. Thank you both for the reports!
